CHANGELOG.md

```diff
@@ -4,6 +4,30 @@ All notable changes to this project will be documented in this file.
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## [0.3.4] - 2023-01-10
+
+We release version 0.3.4 which you can install with `pip install syne-tune[extra]`.
+
+Thanks to all contributors (sorted by chronological commit order):
+@mseeger, @aaronkl, @jgolebiowski, @geoalgo, @wistuba
```

Comment:
[diffing v0.3.3 against main](https://github.com/awslabs/syne-tune/compare/v0.3.3...main) results in the below; would this be the most up-to-date for v0.3.4? (Just including the commits after 0.3.3.)

```suggestion
@wesk, @mseeger, @610v4nn1, @aaronkl, @jgolebiowski
```

Reply:
Uh sorry, I did not do this properly. Sorry for not adding you. We should maybe add everybody on the team, as they help with PRs and stuff.